source/manual/monitor-sidekiq-workers.html.md

```diff
@@ -22,6 +22,16 @@ Then visit [http://127.0.0.1:9000](http://127.0.0.1:9000) to see a list of [Side
 
 You can also monitor Sidekiq queue lengths using [this Grafana dashboard](https://grafana.publishing.service.gov.uk/dashboard/file/sidekiq.json).
 
+To view your local Sidekiq queue, go to the [sidekiq-monitoring app](https://github.com/alphagov/sidekiq-monitoring) in the vm and run `./bin/foreman start` for all applications, or `./bin/foreman run <app_name>` for a specific app.
+
+Make a note of the port number of the app whose queue you want to view. Then in your local machine, go to govuk-puppet/development-vm and run:
```

Comment:
Is there somewhere you can look this up? Puppet?

Also, is it definitely the port number of the app? I would have guessed that it would be the port number for something to do with Sidekiq, but I don't really understand how it all hooks together. :)